parm/config/config.anal

```diff
@@ -117,12 +117,12 @@ if [[ $RUN_ENVIR == "emc" ]]; then
 
 #   Assimilate DO-2 GeoOptics
     if [[ "$CDATE" -ge "2021031712" && "$CDATE" -lt "2021091612" ]]; then
-        export CONVINFO=$FIXgsi/gfsv16_historical/global_convinfo.txt.2021031712
+        export CONVINFO=$FIXgsi/gfsv16_historical/gfsv16.3/global_convinfo.txt.2021031712
```

Comment:
Dogwood directory `/lfs/h2/emc/da/noscrub/emily.liu/git/global-workflow/gfs.v16.3.0/sorc/gsi.fd/fix/gfsv16_historical/gfsv16.3` does not contain `global_convinfo.txt.2021031712`.   This directory contains `global_convinfo.txt.2021091612` and `global_convinfo.txt.2022031612`.